src/account/handle.rs

```diff
@@ -63,6 +63,16 @@ impl AccountHandle {
         self.read().await.alias.clone()
     }
 
+    /// Get the [`OutputData`] of an output stored in the account
+    pub async fn get_output(&self, output_id: &OutputId) -> Result<OutputData> {
```

Comment:
I would expect things like that to return an Option

Reply:
Yes makes more sense, I just thought it could be weird for the bindings, because in other languages it's not so clear
But this can also be handled there in the binding side, will change it